package.json

```diff
@@ -41,7 +41,6 @@
     "lodash.reduce": "^4.6.0",
     "lodash.some": "^4.6.0",
     "lodash.values": "^4.3.0",
-    "rxjs": "^5.0.0-beta.12",
-    "rxjs-serial-subscription": "^0.1.1"
+    "rxjs": "^5.0.0-beta.12"
```

Comment:
you may want to upgrade into latest public, which also fixes leaks inside of implementation.